.github/workflows/upload-binary.yml

```diff
@@ -0,0 +1,70 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Upload Binary
+
+on:
```

Comment:
Why trigger on push tags? Not release created or published?

```
on:
  release:
    types: [created]
```

Reply:
I got this reference from https://github.com/ariya/berkala/blob/main/.github/workflows/release.yml

I tested with release: types: [created] too in my personal repo here: https://github.com/dennypradipta/discord-bot/releases/tag/v.0.10 and it works, too.

Which one do you think I should use?

Reply:
I think `on release published` is your purpose. Since the tags can be created without release :)

Reply:
Tested in this https://github.com/dennypradipta/discord-bot/releases/tag/v0.11

Cheers!